runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -12081,7 +12616,24 @@ J9::X86::TreeEvaluator::directCallEvaluator(TR::Node *node, TR::CodeGenerator *c
          else
             break;
          }
-
+      case TR::java_lang_StringCoding_countPositives:
+         {
+         if (comp->target().cpu.supportsFeature(OMR_FEATURE_X86_SSE2) && feGetEnv("replaceCountPositives") != NULL)
```

Comment:
You already called `cg->setSupportsInlineStringCodingCountPositives()` so you should be calling `cg->getSupportsInlineStringCodingCountPositives()`

Reply:
I consolidated all of the inlining logic into `InlinerTempForJ9`, since I was already using it to conditionally only inline `countPositives` into `hasNegatives`. I've confirmed that the inlining behaviour is still correct using this method.